ports/raspberrypi/boards/waveshare_rp2040_one/mpconfigboard.mk

```
@@ -0,0 +1,11 @@
+USB_VID = 0x2E8A
+USB_PID = 0x103A
+USB_PRODUCT = "RP2040-One"
+USB_MANUFACTURER = "Waveshare Electronics"
+
+CHIP_VARIANT = RP2040
+CHIP_FAMILY = rp2
+
+EXTERNAL_FLASH_DEVICES = "W25Q32JVSSIQ"
```

Use this name for the flash chip. See https://github.com/adafruit/nvm.toml/blob/main/flash/winbond/W25Q32JVxQ.toml
```suggestion
EXTERNAL_FLASH_DEVICES = "W25Q32JVxQ"
```

---

All checks now pass